theme/templates/resource-landing-page/citation.html

```diff
@@ -21,92 +21,129 @@
-                                            </div>
-                                        </div>
-                                        <div id="div_id_url" class="control-group">
-                                            <span for="id_url">Url</span>
+                                    <div class="controls">
+                                            <textarea class="form-control input-sm textarea" cols="40"
+                                                      id="id_statement" name="statement" readonly="True">{{ cm.metadata.rights.statement }}</textarea>
+                                    </div>
+                                </div>
+                                <div id="div_id_url" class="control-group">
+                                    <span for="id_url">Url</span>
 
-                                            <div class="controls">
-                                                <input class="form-control input-sm urlinput" id="id_url"
-                                                       maxlength="200" name="url" readonly="True" type="url"
-                                                       value={{ cm.metadata.rights.url }}>
-                                            </div>
-                                        </div>
+                                    <div class="controls">
+                                        <input class="form-control input-sm urlinput" id="id_url"
+                                               maxlength="200" name="url" readonly="True" type="url"
+                                               value={{ cm.metadata.rights.url }}>
+                                    </div>
+                                </div>
 
-                                <button type="button" class="btn btn-primary pull-right btn-form-submit"
-                                        style="display: none;">Save changes
-                                </button>
-                            </div>
+                        <button type="button" class="btn btn-primary pull-right btn-form-submit"
+                                style="display: none;">Save changes
+                        </button>
+                    </div>
                         </fieldset>
                     </form>
                 </div>
             {% endif %}
         </div>
+        <div class="col-xs-12">
+            {#   Require license agreement for file download flag    #}
+            {% if is_owner_user %}
+                <form class="pull-left" action="/hsapi/_internal/{{ cm.short_id }}/set-resource-flag/"
+                      method="POST">
+                    {% csrf_token %}
+
+                    <input name="t" type="hidden" {% if cm.raccess.download_agreement %}value="make_not_require_lic_agreement"
+                           {% else %}value="make_require_lic_agreement"{% endif %}/>
```

Comment:
The existing view function expects this input name to be ```t``` which is used for setting all other resource flags like public/private/discoverable.